src/players/Vidyard.js

```diff
@@ -19,6 +19,10 @@ export default class Vidyard extends Component {
     const { config, onError, onDuration } = this.props
     const id = url && url.match(MATCH_URL_VIDYARD)[1]
     getSDK(SDK_URL, SDK_GLOBAL, SDK_GLOBAL_READY).then(Vidyard => {
+      if (this.player) {
```

Comment:
I put this chunk before `if (!this.container) return` because it gives more explicit behaviour. For example:
- User watches video and he clicks link to open another video.
- If container check will be before this chunk and container is missing (somehow), then new video isn't opened and previous video will continue to play. Nothing happens for user.
- If destroy happens before and no container, then previous video will be stop and new video isn't opened. And for user it's more explicit that problem with video playing, but not with link to open video.